hack/verify-codecgen.sh

```diff
@@ -45,15 +45,19 @@ function cleanup {
 }
 trap cleanup EXIT
 
-# build codecgen tool
+# Build codecgen from Godeps.
+# However, we need to install godep first.
 # We make some tricks with GOPATH variable to make it work with Travis.
 _gopath=${GOPATH}
 export GOPATH="${_tmpdir}"
-go get -u github.com/ugorji/go/codec/codecgen 2>/dev/null
-go install github.com/ugorji/go/codec/codecgen 2>/dev/null
-CODECGEN="${_tmpdir}/bin/codecgen"
+go get -u github.com/tools/godep 2>/dev/null
+go install github.com/tools/godep 2>/dev/null
```

Comment:
also, Travis, Shippable, and Jenkins all install godep before running any checks. So this may not be strictly necessary here?